challenge.js

```diff
@@ -10,6 +10,10 @@
 
 // ANSWER
 
+    // Using browser tools, I would inspect the element I wanted to change to get it's ID.  I would create a variable that looks up the element by ID, and call it.  Then I would run the expression below adding new text to the sting at the end.
+
+      // document.getElementById("ID").innerHTML = "New text!";
+
```

Comment:
One right approach! The only issue is that the question wants us to make a new HTML element, and not just create new text. But that can be done by simply writing new HTML inside the string, e.g.
`"<span> Some new text here. </span>"`